server/src/main/java/org/apache/druid/segment/handoff/CoordinatorBasedSegmentHandoffNotifier.java

```diff
@@ -128,7 +128,7 @@ void checkForSegmentHandoffs()
     catch (Throwable t) {
       log.error(
           t,
-          "Exception while checking handoff for dataSource[%s], Will try again after [%d]secs",
+          "Exception while checking handoff for dataSource[%s], Will try again after [%d]ms",
```

Comment:
```suggestion
          "Exception while checking handoff for datasource[%s]; will try again after [%s]",
```

Reply:
Thanks for the review @abhishekrb19 Just pushed an update